CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@
 
 The Code of Conduct explains the *bare minimum* behavior
 expectations the Node Foundation requires of its contributors.
-[Please read it before participating](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md).
+[Please read it before participating](https://github.com/nodejs/admin/blob/main/CODE_OF_CONDUCT.md).
```

Comment:
Totally optional nit: To future-proof these, I believe you could use `HEAD`?

```suggestion
[Please read it before participating](https://github.com/nodejs/admin/blob/HEAD/CODE_OF_CONDUCT.md).
```

If you do that, these particular changes can land before the name change because they will work before or after the name change.